integration/src/test/java/org/mule/shutdown/ShutdownAppInDomainTestCase.java

```diff
@@ -126,8 +126,8 @@ public void httpRequest() throws IOException, TimeoutException {
   public void jms() throws MuleException {
     final MuleContext muleContextForApp = getMuleContextForApp("app-with-flows");
 
-    muleContextForApp.getClient().dispatch("jms://in?connector=sharedJmsConnector", of("payload"));
-    muleContextForApp.getClient().request("jms://out?connector=sharedJmsConnector", MESSAGE_TIMEOUT);
+    //muleContextForApp.getClient().dispatch("jms://in?connector=sharedJmsConnector", of("payload"));
```

Comment:
remove commented code

Reply:
Test are ignored due to MULE-10335, they need to be fixed when that issue is resolved (by the way, the issue is marked as resolved but the tests are still ignored).